packages/clickable/tests/clickable.test.tsx

```diff
@@ -73,12 +73,10 @@ describe("Clickable", () => {
           <span>default tabIndex</span>
         </Clickable>
       );
-      expect(
```

Comment:
This was breaking tests, even in master. Maybe related to some recent package upgrades?